src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ParameterIdentificationEngine.cs

```diff
@@ -68,8 +69,9 @@ await Task.Run(() => Parallel.ForEach(parameterIdentificationRuns, parallelOptio
          }
          finally
          {
-            parameterIdentificationRuns.Each(x => x.Dispose());
+            _parameterIdentificationRuns.Each(x => x.Dispose());
             _eventPublisher.PublishEvent(new ParameterIdentificationTerminatedEvent(parameterIdentification));
+            _parameterIdentificationRuns = null;
```

Comment:
that is not requiired. However, `_parameterIdentificationRuns.Clear` is required IMO